test/common/plugin/currencyPlugins.spec.ts

```diff
@@ -12,7 +12,7 @@ import {
 } from 'edge-core-js/types'
 
 import fixtures from '../../../fixtures/currencyPlugins.json'
-import plugin from '../../../src/btc'
+import plugin from '../../../src'
```

Comment:
Still needs to be `/index`. The Node behavior of auto-guessing "index" is non-standard.